parts/kuberneteskubelet.service

```diff
@@ -45,6 +45,7 @@ ExecStart=/usr/bin/docker run \
         --azure-container-registry-config=/etc/kubernetes/azure.json \
         --hairpin-mode=promiscuous-bridge \
         --network-plugin=${KUBELET_NETWORK_PLUGIN} \
+        --node-status-update-frequency=1m \
```

Comment:
TODO: enable configurable `node-status-update-frequency`, with a default value of `10s` if unspecified